### DISCUSSION TITLE
[@mantine/dates] Term `column` is overloaded

### DISCUSSION THREAD

**Discussion by filiptammergard** [mantinedev/mantine#3927] [Feedback]
Hey!

Both when looking into documentation and when doing some code contributions, I noticed that the term `column` is overloaded when it comes to the date components which makes things a bit confusing at times.

We have the props `columnsToScroll` and `numberOfColumns` in many date components. In this context, `column` refers to the entire picker.

On the other hand, in the keyboard interactions section, it says that ArrowDown focuses next year/month/date "in a column that is not disabled". In this context, `column` refers to the table grid column within a date picker.

Attaching some diagrams of terminology used.

DatePicker:
![image](https://user-images.githubusercontent.com/44197016/228176650-f4755bd2-70ba-4ba6-b344-330aa23e5017.png)

MonthPicker:
![image](https://user-images.githubusercontent.com/44197016/228176707-bdf52b40-ca22-422e-b319-031fe0b93ebd.png)

YearPicker:
![image](https://user-images.githubusercontent.com/44197016/228176789-9f50b261-b629-4f9e-9e69-5b4e20152641.png)

Hopefully that makes it clear what I mean by `column` being overloaded as a term. It's the same in code that `column` in different places refer to different things.

For me, it makes sense to reserve the term `column` for the semantic meaning in the context of a table grid, which makes it simple to use together with the terms `row` and `cell`. 

I'd consider changing the term for the other meaning to `picker`, which would mean changing the prop names to `pickersToScroll` and `numberOfPickers`. I know this is a breaking change, so I'm mostly looking for discussion and see if this is something that could be planned for v7.

What do you think?

**Comment by rtivital** [comment]
I'll consider renaming it in v7, though it is too far away now

**Comment by filiptammergard** [comment]
Seems like v7 is on its way. Is this something you consider @rtivital?

**Reply by rtivital** [reply]
I've decided that it is fine as is and pretty clear to everyone
